products/firefox_desktop/testing.md

```diff
@@ -61,4 +69,4 @@ Download the installer version and make sure that everything is working correctl
 Photon is the redesigned UI that will ship in Firefox 57:
 * The hamburger menu will be replaced by a static dropdown menu. If you’re on Nightly and still see the old menu, make sure that `browser.photon.structure.enabled` is set to `true` in about:config (that’s the default value in Nightly 56 and later).
 * Test the *Overflow menu*: the `>>` symbol will appear automatically in the main toolbar when resizing the window to a small width. You can then pin an element to it, or add/remove items from the toolbar directly from their context menu.
-* Test the onboarding tour by clicking the small fox displayed in the left corner on the New tab. There are also onboarding notifications, displayed at the bottom of the page: you can cycle through them by opening multiple new tabs in a row. Note that the onboarding is currently not compatible with Activity Stream.
+* Test the onboarding tour by clicking the small fox displayed in the left corner on the New tab. There are also onboarding notifications, displayed at the bottom of the page: you can cycle through them by opening multiple new tabs in a row.
```

Comment:
the New tab *page*?

Reply:
You mean "at the bottom of the (New tab) page"? I thought it's implicit from the previous sentence, by I don't mind repeating it.

Reply:
or "left corner of the New tab" + "page"?

Reply:
Yeah, either way, it just sounded weird, but that may be just me